taca/storage/cli.py

```diff
@@ -0,0 +1,34 @@
+""" CLI for the storage subcommand
+"""
+import click
+from taca.storage import storage as st
+
+
+@click.group()
+@click.option('-d', '--days', type=int, default=2,
+		      help="Days to consider a run old")
+@click.option('-r', '--run', type=click.Path(exists=True))
+@click.pass_context
+def storage(ctx, days, run):
+	""" Storage management methods and utilities """
+	pass
+
+# Storage subcommands
+@storage.command()
+@click.option('--backend', type=click.Choice(['swestore']), required=True,
```

Comment:
@ingkebil this should easy integration if you guys want to add another storage backend (i.e tape)